src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/ConfiguredProjectImplicitActivationTracking.cs

```diff
@@ -1,81 +1,52 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
 
-using System;
-using System.Collections.Generic;
 using System.ComponentModel.Composition;
-using System.Linq;
-using System.Threading.Tasks;
-using System.Threading.Tasks.Dataflow;
 
 namespace Microsoft.VisualStudio.ProjectSystem
 {
     /// <summary>
     ///     Responsible for activating and deactiving <see cref="IImplicitlyActiveService"/> instances.
     /// </summary>
-    internal class ConfiguredProjectImplicitActivationTracking : OnceInitializedOnceDisposed
+    [Export(ExportContractNames.Scopes.ConfiguredProject, typeof(IProjectDynamicLoadComponent))]
+    [AppliesTo(LoadCapabilities)]
+    internal partial class ConfiguredProjectImplicitActivationTracking : AbstractMultiLifetimeComponent<ConfiguredProjectImplicitActivationTracking.ConfiguredProjectImplicitActivationTrackingInstance>, IProjectDynamicLoadComponent
     {
+        // NOTE: Ideally this component would be marked with 'AlwaysApplicable' so that we always load
+        // IImplicitlyActiveService instances in all project types regardless of exported capabilities,
+        // but doing so would cause us to be loaded in lots of situations even when not needed. Instead,
```

Comment:
```suggestion
        // but doing so would cause the .NET Project System's assemblies to be loaded in lots of situations even when not needed. Instead,
```